src/ORM/AssociationCollection.php

```diff
@@ -15,6 +15,7 @@
 namespace Cake\ORM;
 
 use ArrayIterator;
+use Cake\Datasource\AssociationsNormalizerTrait;
```

Comment:
I was trying to comment on the AssociationCollection as a whole. The ORM version is still there.